caluma/caluma_form/filters.py

```diff
@@ -64,8 +65,67 @@ class Meta:
         fields = ("meta", "attribute")
 
 
+class VisibleOptionFilter(Filter):
+    """
+    Lookup type display options in the context of a document.
+
+    This will make sure all the `is_hidden`-JEXLs on the options are evaluated in the
+    context of the provided document.
+
+    Note:
+    This filter can only be used if the options in the QuerySet all belong only to
+    one single question. Generally forms are built that way, but theoretically,
+    options could be shared between questions. In that case it will throw a
+    `ValidationError`.
+
```

Comment:
```suggestion
    Note:
    This filter can only be used if the options in the QuerySet all belong only to
    one single question. Generally forms are built that way, but theoretically,
    options could be shared between questions. In that case it will throw a
    `ValidationError`.

    Also note that this evaluates JEXL for all the options of the question, which
    has a good bit of performance impact.
```

Reply:
Done :+1: